std/net/isemail.d

```diff
@@ -26,9 +26,14 @@ module std.net.isemail;
 
 // FIXME
 import std.range.primitives; // : ElementType;
+import std.regex;
 import std.traits;
 import std.typecons : Flag, Yes, No;
 
+private enum ipRegex = ctRegex!(`\b(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}`~
+                        `(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$`);
```

Comment:
Please no enums with ctRegex, it degrades performance because that would copy-paste initialization to the place of use.

Use auto or static.